.vscode/settings.json

```diff
@@ -1,7 +1,6 @@
 {
   "typescript.tsdk": "node_modules/typescript/lib",
   "editor.tabSize": 2,
-  "editor.formatOnSave": true,
```

Comment:
I think this was a mistake after having added the prettier formatter into the workspace project (which we thought made sense) but while removing it @elanbibas may have inadvertently removed this one too. So will add it back. But re the formatter setting, since you guys use prettier to validate the PRs, wouldn't it make sense to have it on the workspace settings too?

Reply:
Isn't that file the configuration for the current workspace ? We do use auto format on save (for prettier and tslint)

Reply:
yeah, this was removed by mistake and will be added again, was just asking because we also added the prettier formatter setting in there and were asked to remove it. So wondering if all other developers need to set that in user settings instead of workspace settings for this to work.